qutebrowser/misc/utilcmds.py

```diff
@@ -247,3 +247,34 @@ def log_capacity(capacity: int):
         raise cmdexc.CommandError("Can't set a negative log capacity!")
     else:
         log.ram_handler.change_log_capacity(capacity)
+
+
+@cmdutils.register(debug=True, name='debug-log-level')
```

Comment:
No need to set `name=` anymore as the name is now automatically `debug-log-level` when the function is named `debug_log_level`.